debian/control

```diff
@@ -17,8 +17,8 @@ X-Python3-Version: >= 3.4
 Package: qubes-desktop-linux-common
 Architecture: any
 Depends:
- python3-qubesimgconverter,
```

Comment:
I'd prefer adding Provides to qubes-utils package instead (if it works in Debian). That's because we may want to split that part into separate subpackage (similar to rpm one) at some point. Or maybe go ahead and split it now already.

Reply:
https://github.com/QubesOS/qubes-linux-utils/pull/58